### PR TITLE
Show transmission rate in status bar in mainwindow

### PR DESCRIPTION
**Pull request by krf**

### CODE REVIEW

**Comment by krf** [comment]
gammaray-transmissionrate-in-mainwindow

**Comment by krf** [comment]
Comments? Much more visible than in the debug output IMO. Could be enabled conditionally by an environment variable.

**Comment by vkrause** [comment]
I agree. Only useful for GammaRay developers though, not for most of the users, so hiding this by default is probably a good idea indeed.

**Comment by krf** [comment]
Needs the following env var now: `GAMMARAY_DEVELOPERMODE=1`

File: common/endpoint.h [hunk 1]
```diff
@@ -255,7 +257,8 @@ private slots:
 
     QPointer<QIODevice> m_socket;
     Protocol::ObjectAddress m_myAddress;
-    int m_bytesTransferred;
+    int m_bytesRead;
+    int m_bytesWritten;
```
Comment (vkrause) [inline common/endpoint.h]: uint64 while we are at it please, we already managed to overflow those counters

File: ui/mainwindow.cpp [hunk 2]
```diff
@@ -313,6 +313,14 @@ MainWindow::MainWindow(QWidget *parent)
     connect(UiIntegration::instance(), SIGNAL(navigateToCode(QUrl,int,int)), this,
             SLOT(navigateToCode(QUrl,int,int)));
 
+    const bool developerModeEnabled = !QString::fromLocal8Bit(qgetenv("GAMMARAY_DEVELOPERMODE")).isEmpty();
```
Comment (vkrause) [inline ui/mainwindow.cpp]: Nice, @chsterz we probably want to use that for the remote view FPS option too. Same for the message statistics view.

**Pull request closed without merging**